episodes/itksnap.Rmd

```diff
@@ -0,0 +1,630 @@
+---
+title: 'itksnap'
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to understand and visualise volumetric medical data in ITK-SNAP?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Medical imaging data in DICOM and nifti formats
+- Getting started with ITK-SNAP
+- Viewing data
+- Converting data
+- Modifyiing data
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Medical Imaging Data 
+In these exercises we will be working with real-world medical imaging data from the Cancer Imaging Archive (TCIA).
+
+### Data resources
+
+#### CT-vs-PET-Ventilation-Imaging 
+CT Ventilation as a Functional Imaging Modality for Lung Cancer Radiotherapy from [TCIA](https://www.cancerimagingarchive.net/collection/ct-vs-pet-ventilation-imaging/).
+We recomend to focus on exhale/inhale breath hold CT (BHCT) Dicoms which belog to: 
+* The BHCT scans for CT-PET-VI-02 & CT-PET-VI-03 show very little motion between inhale and exhale
+* The BHCT scans for CT-PET-VI-05 have a different number of slices between the inhale and exhale
+
+Data paths look like these:
+```
+/CT-PET-VI-02$ tree -d 
```

Comment:
I get 
> Command 'tree' not found
on linux.
Need to check dependencies for all OS

Reply:
[74e2376](https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/pull/5/commits/74e2376b1141f9ce40a3f8dd34782bfc1d11fe7e)